src/OSPSuite.Core/Domain/CoreOriginData.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public class CoreOriginData : BaseOriginData
```

Comment:
I don't get the baseOriginData, vs CoreOriginData.
Why do we need this?

Reply:
The only thing that exists in CoreOriginData is Species as a string. The rest is all the same. In Core we want to be able to set and get that string, but not in PKSim (not as a string anyway)

Reply:
generic would probably work here. TSpecies or something like that.